harvard-lts/CURIOSity
# Node healthcheck

**Adds node healthcheck (merge after ssl removal)**
* * *

# What does this Pull Request do?
Switches to a node health check script for internal health check

# How should this be tested?

You could stick the new health check in your local docker-compose, or try this on dev. New health check command is:
- "/opt/healthcheck/healthcheck http://localhost:4000/healthcheck"

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
